### PR TITLE
Update styles add token to support darkMode

### PR DESCRIPTION
**Pull request by biocom**
Addresses #983

Screen Shot 2021-07-21 at 18 13 45Screen Shot 2021-07-21 at 18 13 34

### CODE REVIEW

File: src/custom/components/SearchModal/CurrencyList/CurrencyListMod.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ import { MouseoverTooltip } from 'components/Tooltip'
 import { MenuItem } from 'components/SearchModal/styleds'
 import Loader from 'components/Loader'
 import { isTokenOnList } from 'utils'
-import ImportRow from 'components/SearchModal/ImportRow'
+import ImportRow from 'components/SearchModal/CurrencyList/ImportRow'
```
Comment (W3stside) [inline src/custom/components/SearchModal/CurrencyList/CurrencyListMod.tsx:20]: this one is a typo no? shouldn't it be ImportRow from it's own folder?

Comment (W3stside) [reply src/custom/components/SearchModal/CurrencyList/CurrencyListMod.tsx:20]: move `ImportRow` to it's own thing like `components/SearchModal/ImportRow` where inside there it is:
```
ImportRow /
------- ImportRowMod.tsx     <---- UniCode and some mods
------- index.tsx
```

**Review by alfetopito — APPROVED** [review]
Issue is fixed :+1:

**Comment by alfetopito** [comment]
> Addresses #983
> 
> Screen Shot 2021-07-21 at 18 13 45Screen Shot 2021-07-21 at 18 13 34

I can't help but notice that the button says `Import Test`.

Since when is that there?
I don't think it should say `Test`

File: src/custom/components/SearchModal/CurrencyList/ImportRow/ImportRowMod.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+          width="fit-content"
+          padding="6px 12px"
+          fontWeight={500}
+          fontSize="14px"
+          onClick={() => {
+            setImportToken && setImportToken(token)
+            showImportView()
+          }}
+        >
+          <Trans>Import Test</Trans>
```
Comment (W3stside) [inline src/custom/components/SearchModal/CurrencyList/ImportRow/ImportRowMod.tsx:94]:
```suggestion
          <Trans>Import</Trans>
```

File: src/custom/components/SearchModal/CurrencyList/CurrencyListMod.tsx (hunk 1 above)
Comment (W3stside) [reply src/custom/components/SearchModal/CurrencyList/CurrencyListMod.tsx]: done in https://github.com/gnosis/cowswap/pull/1005/commits/a93783b0754d912604ef095cbe0d45064e0a53b7

File: src/custom/components/SearchModal/CurrencyList/ImportRow/ImportRowMod.tsx (hunk 2 above)
Comment (alfetopito) [reply src/custom/components/SearchModal/CurrencyList/ImportRow/ImportRowMod.tsx]: :clap:

**Comment by W3stside** [comment]
@biocom @alfetopito I made slight changes to this which can easily be adapted now. I pushed directly here apologies. 

I modded the ManageTokens comp and set the colours more inline with what darkmode has in other places, but lke i said, now that it's modded it can easily be changed in the file.

5b359e5

**Review by W3stside — APPROVED** [review]

**Comment by biocom** [comment]
> > Addresses #983
> > Screen Shot 2021-07-21 at 18 13 45Screen Shot 2021-07-21 at 18 13 34
> 
> I can't help but notice that the button says `Import Test`.
> 
> Since when is that there?
> I don't think it should say `Test`

The 'test' part was definitely something I forgot to cleanup 👀

**Comment by biocom** [comment]
@W3stside Thanks for the fixes, think I went overzealous with the replace path. Looks good to me 👍🏼

**Comment by alfetopito** [comment]
Double approval :+1: :+1:

**Pull request merged**